builtin/logical/rabbitmq/path_config_connection.go

```diff
@@ -91,6 +100,33 @@ func (b *backend) pathConnectionUpdate(ctx context.Context, req *logical.Request
 	return nil, nil
 }
 
+func readConfig(ctx context.Context, storage logical.Storage) (config connectionConfig, err error) {
```

Comment:
nit: It doesn't seem like the named parameters are being used in this function, maybe they're not necessary?

```suggestion
func readConfig(ctx context.Context, storage logical.Storage) (connectionConfig, error) {
```